module/helpers.ts

```diff
@@ -206,6 +206,9 @@ export type StringIndexedObject<T> = { [i: string]: T };
 /** For Sorting Items/Actors/Etc. by Name */
 export const byName = (a: { name: string }, b: { name: string }): number => a.name.localeCompare(b.name);
 
+/** For removing quotes and apostrophes from strings */
+export const escapeQuotes = (a: string): string => a.replace(/\\([\s\S])|(['"])/g, "\\$1$2");
```

Comment:
Regexes are always quite impenetrable to try and parse, but looks like it's covering some interesting cases.

:+1:

is `[\s\S]` not equivalent to `[.]` though?

Reply:
\\([\s\S]) will find any character that is already escaped, including newline (I imagine we don't want to try to escape them twice). If so the escaped character is replaced by itself with \\$1.
Otherwise (['"]) will find any un-escaped quotes or apostrophes and replace it with \\$2.
$1 and $2 will never be used at the same time due to the |.

This can be expanded later if we find that other special characters cause issues in the future.